app/controllers/results_controller.rb

```diff
@@ -12,6 +12,7 @@ def index
   def show
     @results = AnswerSet.find_by(uuid: params[:uuid])
     @all_other_results = AnswerSet.last(1000).reject{ |r| r.id == @results.id }.pluck(:x_axis_scaled, :y_axis_scaled)
+    @locale = I18n.locale
```

Comment:
@drewpotter I meant the other way around, I dont think we need to pass this in at all from the front end, we should just be able to grab it in the filter logic in the controller

Reply:
@spencerldixon I don't completely understand this. I think the params for the locale are not being sent when the filters are being selected. It's like it drops them off. This is why I added a hidden field to append those.